### ISSUE TITLE
Failing test o.e.j.websocket.server.misbehaving.MisbehavingClassTest

### ISSUE THREAD

**Issue by janbartel** [eclipse/jetty.project#2598]
https://jenkins.webtide.net/job/jetty.project/job/jetty-9.4.x/358/testReport/junit/org.eclipse.jetty.websocket.server.misbehaving/MisbehavingClassTest/linux_jdk8___Test___jdk8___testAnnotatedRuntimeOnConnect/

**Comment by joakime** [comment]
Duplicate of #2567

**Comment by janbartel** [comment]
Nope, its not a duplicate.  #2567 is for stack traces that appear on output. This is an actual test failure.

**Comment by joakime** [comment]
The assertion reported in CI, copied here for long term reference.

```
closeStatusCode
Expected: is <1011>
     but: was <-1>
	at org.eclipse.jetty.websocket.server.misbehaving.MisbehavingClassTest.testAnnotatedRuntimeOnConnect(MisbehavingClassTest.java:142)
```

The assertion error code reference permlink, also for long term reference ...

https://github.com/eclipse/jetty.project/blob/1509d1c9aa97e108de7d978f6ac68668e258db16/jetty-websocket/websocket-server/src/test/java/org/eclipse/jetty/websocket/server/misbehaving/MisbehavingClassTest.java#L140-L142
